docs/human-interface-guidelines.md

```diff
@@ -785,6 +784,14 @@ A Static Notebook is a small set of unchanging tabs, commonly seen in preference
 
 A Dynamic Notebook is a way for an app to provide user-manageable tabbing functionality, commonly seen in web browsers. The tabs appear attached to the toolbar on their own tab bar above the relevant content. Tabs are able to be rearranged and closed, and a "new tab" button is at the start of the notebook widget.
 
+## Toasts {#toasts}
+
+A Toast is an ephmeral in-app notification overlaid on top of other content used to affirm user action and/or provide a time-sensitive contextual action. After a short timeout, a Toast is automatically dismissed. Because they are ephemeral, Toasts should follow recent user action when the user is expected to engaging with the app. A Toast can offer a contextual action that can be triggered as long as the Toast is visible. A Toast includes a manual dismiss button, e.g. if it is likely to be covering content the user may be more interested in.
```

Comment:
```suggestion
A Toast is an ephemeral in-app notification overlaid on top of other content used to affirm user action and/or provide a time-sensitive contextual action. After a short timeout, a Toast is automatically dismissed. Because they are ephemeral, Toasts should follow recent user action when the user is expected to engaging with the app. A Toast can offer a contextual action that can be triggered as long as the Toast is visible. A Toast includes a manual dismiss button, e.g. if it is likely to be covering content the user may be more interested in.
```